core/arch/arm/plat-imx/conf.mk

```diff
@@ -16,10 +25,23 @@ $(call force,CFG_GIC,y)
 $(call force,CFG_IMX_UART,y)
 $(call force,CFG_PM_STUBS,y)
 $(call force,CFG_WITH_SOFTWARE_PRNG,y)
-ifeq ($(PLATFORM_FLAVOR),mx6ulevk)
+
+CFG_CRYPTO_SIZE_OPTIMIZATION ?= n
+CFG_WITH_STACK_CANARIES ?= y
+
+
+# i.MX6UL specific config
+ifeq ($(CFG_MX6UL),y)
+arm32-platform-cpuarch		:= cortex-a7
+
 $(call force,CFG_SECURE_TIME_SOURCE_CNTPCT,y)
 endif
-ifeq ($(PLATFORM_FLAVOR),$(filter $(PLATFORM_FLAVOR),mx6qsabrelite mx6qsabresd mx6dlsabresd))
+
+
+# i.MX6 Solo/DualLite/Dual/Quad specific config
+ifeq ($(filter y, $(CFG_MX6Q) $(CFG_MX6D) $(CFG_MX6DL) $(CFG_MX6S)), y)
```

Comment:
As the i.MX6 Solo is not yet supported, such change would only partially allow its support, and maybe miss some required changes. I think it is better to add this config once the platform is fully supported. But that remains my feeling add I let the other iMX6 maintainers (@MrVan @coolfengyu @yanyan-wrs) give their feedback on this.